src/composables/useTask.ts

```
@@ -0,0 +1,85 @@
+export default function useTask() {
+    const store = useTaskStore();
+    const isLoading = ref(false);
+    const DEFAULT_PAGE = 1;
+    const DEFAULT_ITEMS_PER_PAGE = 10;
+    const successMessage = ref('');
+    const fetchTasks = async(page: number = DEFAULT_PAGE, itemsPerPage: number = DEFAULT_ITEMS_PER_PAGE) => {
+        try {
+            isLoading.value = true;
+            await store.fetchTasks(page, itemsPerPage);
+        } catch (err:any) {
+            throw err;
```

Biome (1.9.4)

[error] The catch clause that only rethrows the original error is useless.

An unnecessary catch clause can be confusing.
Unsafe fix: Remove the catch clause.


(lint/complexity/noUselessCatch)